### ISSUE TITLE
How to get the preprocessed data?

### ISSUE THREAD

**Issue by Xyporz** [CaglarGuher/IDRiD-Eye-Fundus-Dataset-Lesion-Segmentation#4]
Nice work! I have downloaded and obtained the data, but I am curious about how I obtained the preprocessed data because I have other data sets that I want to do the same preprocessing.

**Comment by Xyporz** [comment]
I seem to have seen this kind of preprocessed image in another paper, but I can't recall it.

**Comment by CaglarGuher** [comment]
yes there is a paper for the preprocess, however ,that paper's preprocess is working well on the most of the area, except for borders between background of fundus and exudate parts,( I can explain in detail if you want to learn more about it) so the current preprocess is the improved version.The script needs some adjustments as the framework used for the preprocess is changed  and the function no longer works. I can share the fixed function as preprocess_func  folder in couple days.
